manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: istio-pilot{{ .Values.version }}
+  name: istiod{{ .Values.version }}
```

Comment:
I am not sure why there isn't, I didn't want to change what we had since it was already that way. `version` was never widely documented/used/fully implemented, it will be cleaned up with the multiple control plane stuff I am working on soon